src/views/token-swap/components/quote-selector/quote-selector.view.jsx

```diff
@@ -17,7 +17,6 @@ function QuoteSelector({
       <div className={classes.header}>
         <p className={classes.nameHeader}>Liquidity provider</p>
         <p className={classes.toTokensHeader}>You get (ETH)</p>
-        <p className={classes.rewardHeader}>Reward</p>
```

Comment:
this is from token swap, so it should stay. It's' a different kind of reward

Reply:
Thx! Reverted!